src/components/button/Button.css

```diff
@@ -0,0 +1,42 @@
+.button1 {
+    font-size: 16px;
+    background: #171717;
+    border: 3px solid #171717;
+    color: #FFFFFF;
+    border-radius: 7px;
+    padding: 12px 27px;
+    box-sizing: border-box;
+    width: max-content;
```

Comment:
Weten jullie ook wat dit doet? Of heb je die gewoon overgenomen van de echte code? ;)

Reply:
Code aangepast, tevens de arrow een left-margin van 0.5 em meegegeven voor de ruimte.